### PR TITLE
ci: fix wrong debug file name 🐛

### PR DESCRIPTION
**Pull request by ahasna**

### CODE REVIEW

File: .github/workflows/upgrade-test.yml [hunk 1]
```diff
@@ -69,9 +69,6 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Install pnpm
```
Comment (kylezs) [inline .github/workflows/upgrade-test.yml:72]: why is this removed?

**Review by tomjohnburton — APPROVED** [review]
